packages/boilerplate-generator/template-web.browser.js

```diff
@@ -15,17 +26,14 @@ export const headTemplate = ({
   ).join('') + '>',
   '<head>',
 
-  ...(css || []).map(file =>
-    template('  <link rel="stylesheet" type="text/css" class="__meteor-css__" href="<%- href %>">')({
-      href: bundledJsCssUrlRewriteHook(file.url),
-    })
-  ),
-
-  head,
+  (headSections.length === 1)
+    ? [cssBundle, headSections[0]].join('\n')
+    : [headSections[0], cssBundle, headSections[1]].join('\n'),
   dynamicHead,
   '</head>',
   '<body>',
 ].join('\n');
+};
```

Comment:
The indentation looks like a little off here - looks like lines 21 - 35 need to be indented.

Reply:
The entire "return [" block needed to be indented. I also added a couple of blank lines to make it easier to see which lines go together.